### PR TITLE
Block-quote advisory text, to help delineate what's actually supposed

### LINKED ISSUE #14: The README documents contact information twice
There are two hunks of guidance text in the README of this repository about providing guidance.  Can they be consolidated into one?

### PR DESCRIPTION
**Pull request by tkphd**
to be edited and what should simply be cut.

Closes #14, which was hopefully also addressed by #13?

### CODE REVIEW

**Comment by ajnelson-nist** [comment]
It looks like the block-quoting assumes very long text lines are used, instead of source text being constrained to a column count.  See the "Guide to Writing README-style Metadata" link, whose markdown syntax was split across two lines and then broken by the `> ` syntax.

**Comment by wd15** [comment]
Trevor: I think I'm with Alex on this. I'm not sure the block quoting really works. The README.md is not a template in and of itself. It's a description of why this repository exists and isn't intended to be a template. We might want to give some links to really nice examples of READMEs (e.g [this](https://github.com/18F/open-source-guide/blob/18f-pages/pages/making-readmes-readable.md) and [this](https://github.com/matiassingers/awesome-readme)). The entire things needs to rewritten for a new project anyway.

**Comment by tkphd** [comment]
Thanks for the feedback, it's useful. I've reverted the blockquote, and rephrased some of the description. Please take another look.

File: README.md [hunk 1]
```diff
@@ -28,65 +17,82 @@ The key files contained in this repository -- which will also
...
+
+Each repository will contain a plain-text [README file][wk-rdm],
+preferably formatted using [GitHub-flavored Markdown][gh-mdn] and
+named `README.md` (this file) or `README`.
 
-Each repository will contain a [README file][wk-rdm], preferably
-formatted using [GitHub-flavored Markdown][gh-mdn] and named
-`README.md` (this file). It must contain:
+Per the [GitHub ROB][gh-rob] and [NIST Suborder
+1801.02][nist-s-1801-02], your README should contain:
```
Comment (wd15) [inline README.md:29]: The NIST Suborder (nist-s-1801-02) doesn't show up as a link when I view it. (nor does the cornell-meta link).

**Review by wd15 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (tkphd) [reply README.md]: Thanks! Found & fixed several typos, and added the Cornell link.

**Pull request merged**